docs/howtos/layers/tracks.md

```diff
@@ -11,21 +11,61 @@ kernelspec:
 displaying tracks and defining relationships between tracks.
 
-## When to use the tracks layer
+For more information about layers, refer to [Layers at a glance](../../guides/layers).
 
-The tracks layer allows you to display trajectories in `nD+t` while visualizing
-the recent history of the track via a fading tail.
+```{note}
+Tracks layers can be created only programmatically, i.e. in the console, using a
+script, etc. Please refer to [A simple example](#a-simple-example) and use the
+code there to add a tracks layer first, then explore the GUI controls.
+```
+
+## When to use the `tracks` layer
+
+The `tracks` layer allows you to display trajectories in `nD+t` while
+visualizing the recent history of the track via a fading tail.
 
 Each track can have annotations associated with it using the `Tracks.properties`
 dictionary. These properties can be used to set the colors of the tracks.
 
 For example, when displaying tracks of different classes/types, one could
-automatically set color the individual tracks by their respective class/type.
+automatically set the color of the individual tracks by their respective
+class/type.
+
+## GUI controls for the `tracks` layer
+
+* Color by - there is a dropdown for this but at present the only choice is
+  `track_id`.
+* Colormap - choose a colormap from the dropdown. These are explained in
+  [the colormaps section](./surface.md#working-with-colormaps) of
+  _Using the surface layer_.
+* Blending - choose `opaque`, `translucent`, `translucent no depth`, `minimum`
+  or `additive` from the dropdown. Refer to the
+  [Blending layers](blending-layers) section of _Layers at a glance_ for an
+  explanation of each type of blending.
+* Opacity - click and hold the circle on the opacity slider bar and adjust it to
+  any value between 0.00 (clear) and 1.00 (completely opaque).
+* Tail width - adjusting the tail width gives the track the appearance of being
+  narrower or wider. At the minimum value, the track looks like a one-pixel
+  line.
+* Tail length - adjusting the tail length gives the track the appearance of
+  being shorter or longer. At the minimum value, it looks something like a
+  dotted line, at the maximum value it almost looks like an unbroken line.
+* Head length - Adjusting the head length gives the track the appearance of
+  being longer. If it is adjusted to the maximum value, the tracks look like
+  stripes. At the minimum value, the tracks flash across the canvas and
+  disappear before starting again.
+* Tail - check this box to see the tracks. If it is not checked, you will not be
+  able to see the tracks at all.
+* Show ID - check this box to display a previously assigned `track_id` label for
+  each track. Assigning values to `track_id` is explained in
+  [Tracks data](#tracks-data) below.
+* Graph - check this box to display a previously created graph as explained in
+  [](#arguments-of-view_tracks-and-add_tracks).
 
 ## A simple example
```

Comment:
I think I would prefer this section about the `GUI controls...`, so right after `When to use...`
I think it makes the GUI controls more clear when you can see the example in the video.?

Reply:
Let me know if this is what you meant.